shared_resources/json/en.lproj/balance_section_step.json

```diff
@@ -1,103 +1,76 @@
-        },
-        "actions": {
-            "goForward": {
-                "type": "default",
-                "buttonTitle": "Putting it in my pocket"
-            }
-        }
-    },
-    {
-        "identifier": "countdown",
-        "type": "countdown",
-        "subtitle": "Begin in...",
-        "fullInstructionsOnly": true,
-        "image": {
-            "type": "fetchable",
-            "imageName": "balance_1",
-            "placementType": "fullsizeBackground"
-        },
-        "duration": 5,
-        "commands": ["playSoundOnStart", "transitionAutomatically", "speakWarningOnPause"],
-        "spokenInstructions": {
-            "start": "Place the phone in your pocket."
-        }
-    },
-    {
-        "identifier": "motion",
-        "type": "balance",
-        "title": "Turn around and then stand still",
-        "duration": 30,
-        "image": {
-            "type": "fetchable",
-            "imageName": "balance_1",
-            "placementType": "fullsizeBackground"
-        },
-        "commands": ["transitionAutomatically", "shouldDisableIdleTimer", "vibrate", "playSound"],
-        "actions": {
-            "skip": {
-                "type": "navigation",
-                "skipToIdentifier": "countdown",
-                "buttonTitle": "Restart test"
-            },
-            "reviewInstructions": {
-                "type": "navigation",
-                "skipToIdentifier": "turnAroundInstructions",
-                "buttonTitle": "Review instructions"
-            }
-        },
-        "spokenInstructions": {
-            "start": "Turn around in a full circle, then stand still, with your feet about shoulder-width apart. Rest your arms at your side and try to not move.",
-            "halfway": "Halfway done.",
-            "23": "Almost done.",
-            "end":"Great job, you are done with this task."
-        }
-    }
-    ]
+    "steps": [
+                {
+                    "identifier": "turnAroundInstructions",
+                    "type": "instruction",
+                    "title": "Turn around, and then stand still",
+                    "detail": "You’ll turn around in a circle and stand still, with your feet shoulder-width apart.\n\nRest your arms by your side and try to avoid moving for 30 seconds.",
+                    "fullInstructionsOnly": true,
+                    "image": {
+                        "type": "animated",
+                        "imageNames": ["balance_1", "balance_1", "balance_1", "balance_2", "balance_3", "balance_4", "balance_5", "balance_6", "balance_7", "balance_8", "balance_9", "balance_10", "balance_11"],
+                        "placementType": "topMarginBackground",
+                        "animationRepeatCount": 1,
+                        "animationDuration": 6.5
+                    },
+                    "actions": {
+                        "goForward": {
+                            "type": "default"
+                        }
```

Comment:
Just remove the button. A default button should always have either a title or icon. Same goes for other cases where you are just deserializing an empty button.